js/webform_civicrm_options.js

```diff
@@ -56,6 +56,10 @@ if (typeof jQuery.fn.prop !== 'function') {
         }
       }).change();
 
+     //Currently we are looking at using #tableSelect thus, we are commenting out 'select-all-civi-options' code for now
+     //until we are certain that we can remove it.
+      
```

Comment:
@aaayyuusshh - indent this one more space and also put a space between so //Current -> // Current and //until -> // until